### PR TITLE
chore: gh action update filter list

### PR DESCRIPTION
**Pull request by kevanantha**
for this discussion -> https://github.com/evilfactorylabs/gatekeeper/discussions/1#discussion-4694082

result -> https://github.com/kevanantha/gatekeeper/actions/runs/3941076532/jobs/6743007185

### CODE REVIEW

File: .github/workflows/update-filter-list.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        with:
+          owner: evilfactorylabs
+          repository: gatekeeper
+      - name: Set up git
+        run: |
+          echo ${{ github.actor }}
+          git config --global user.email "anakmagang@evilfactorylabs.com"
+          git config --global user.name "anakmagang"
+          # git config --global user.name github-actions
+          # git config --global user.email github-actions@github.com
```
Comment (kevanantha) [inline .github/workflows/update-filter-list.yml:23]: not sure ab this @faultables

Comment (faultables) [reply .github/workflows/update-filter-list.yml:22]:
```suggestion
          git config --global user.email "ayang@evilfactorylabs.com"
          git config --global user.name "anakmagang"
```

Comment (faultables) [reply .github/workflows/update-filter-list.yml:22]: maksud gw kek push token or something like that

File: .github/workflows/update-filter-list.yml [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+  workflow_dispatch:
+    schedule:
+      - cron: "0 0 * * SUN"
+
+jobs:
+  update-list:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
```
Comment (faultables) [inline .github/workflows/update-filter-list.yml:11]:
```suggestion
        uses: actions/checkout@v3
        with:
          token: ${{ secrets.PAT }}
```

i guess?

File: .github/workflows/update-filter-list.yml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+      - name: Clone repository
+        uses: GuillaumeFalourd/clone-github-repo-action@v2
+        with:
+          owner: evilfactorylabs
+          repository: gatekeeper
```
Comment (faultables) [inline .github/workflows/update-filter-list.yml:16]:
```suggestion
          repository: gatekeeper
          access-token: ${{ secrets.PAT }}
```

atau ini?

Comment (anakmagang) [reply .github/workflows/update-filter-list.yml:19]: nice let's try

**Review by faultables — APPROVED** [review]
ke surabaya beli vga, lgtm

**Comment by faultables** [comment]
habis sebat gw merge ya, hanupis pisan @kevanantha

**Pull request merged**